LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Userspace Containerization team at Red Hat
```

Comment:
just red hat, we are not a separate enitity :D

Reply:
I've copied that from source-git -- we need to change that also there.

Reply:
It is probably in multiple repos...

Reply:
yup